src/cascadia/TerminalControl/TermControl.xaml

```diff
@@ -351,11 +352,9 @@
                                 <ControlTemplate x:Key="VerticalThumbTemplate"
                                                  TargetType="Thumb">
                                     <Rectangle x:Name="ThumbVisual"
-                                               contract7NotPresent:RadiusX="{Binding Source={ThemeResource ScrollBarCornerRadius}, Converter={StaticResource TopLeftCornerRadiusDoubleValueConverter}}"

```

Comment:
qq: this doesn't matter any longer because we are always running on a contract 7+ machine?

Reply:
Answered for myself! 19H1 is UniversalApiContract _8_, so our checks were _never_ necessary!